### DISCUSSION TITLE
PSADT 3.10.x: Configure Edge extension function

### DISCUSSION THREAD

**Discussion by Pollewops** [PSAppDeployToolkit/PSAppDeployToolkit#1005] [Q&A] [answered]
Just curious.

Why in PSADT 3.10.1 the Edge Extension function uses 'ExtensionSettings' instead of 'ExtensionInstallForceList'.

I see both can be used, but why this decision is made?

**Comment by Pollewops** [comment]
@DanGough you know this?

**Comment by DanGough** [comment]
@seanlillis is the best person to ask regarding this function!

**Reply by Pollewops** [reply]
Ok lets wait for his reply.

**Reply by seanlillis** [reply, marked as answer]
The ExtensionSettings policy works independently of and can take precedence over other policies that you've set elsewhere in group policy or Intune, including the “ExtensionInstallForceList”. As an example, you can have a GPO or Intune policy that uses ExtensionInstallForceList to push mandatory extensions for all devices while alongside that you can push specific extensions to a subset of devices using ExtensionSettings. There are other advantages in using ExtensionSettings including more configuration options. See the doc below for more details. 

https://learn.microsoft.com/en-us/deployedge/microsoft-edge-manage-extensions-ref-guide

**Reply by seanlillis** [reply]
Additionally, ExtensionInstallForceList creates a numbered list of registry keys. A single reg key with JSON string is easier to manage.

**Comment by UliNaatz** [comment]
Für YannickAm 06.11.2024 um 07:53 schrieb Mitch Richters ***@***.***>:﻿
Closed #1005 as resolved.
